### DISCUSSION TITLE
v2.7.0

### DISCUSSION THREAD

**Discussion by tomkerkhove** [kedacore/charts#270] [Announcements]
*Chart: v2.7.0 | App: v2.7.0*

## TL;DR
```shell
helm repo add keda https://kedacore.github.io/charts
helm install keda/keda
```

### Features

- Support for KEDA v2.7.0 ([release notes](https://github.com/kedacore/keda/releases/tag/v2.7.0))
- Support for node selector (https://github.com/kedacore/charts/pull/250)
- Support for seperate Pod Disruption Budgets for Metrics API and operator (https://github.com/kedacore/charts/pull/255 - Contributed by @mac8005 🎉)
- Run secure-by-default as non-root (https://github.com/kedacore/charts/pull/267)

### Fixes / Changes

- Specify supported Kubernetes versions (https://github.com/kedacore/charts/pull/249)

### Breaking Changes

None.

### Removal

None.

## New Contributors
* @v-shenoy made their first contribution in https://github.com/kedacore/charts/pull/263
* @mac8005 made their first contribution in https://github.com/kedacore/charts/pull/255

**Full Changelog**: https://github.com/kedacore/charts/compare/v2.6.2...v2.7.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/v2.7.0'>v2.7.0</a>.</em>
